### DISCUSSION TITLE
Configuring compilation module in nuxt

### DISCUSSION THREAD

**Discussion by jesusarockias** [nuxt/nuxt#9608] [General]
Nuxt is by default using "UMD" script for compilation and I want to know is there any option change the module ES6 module. 

Please share any documentation or let me know where should I change the configuration for this.

**Comment by kissu** [comment]
This is probably what you're looking for:
- https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-build#extend
- https://nuxtjs.org/docs/2.x/features/configuration#extend-webpack-config

Will probably not be able to help more myself tho.

Do you mind sharing with us why you're willing to move out of `UMD`?

**Comment by jesusarockias** [comment]
@kissu Thank you for spending your time to answer my query.

UMD modules will not support tree shaking and it is more comfortable to load ES6 bundled module. I have searched online for any help (github sample, blog post, etc…) but I am not able to find anything.

Can anyone please help me to get the webpack configuration for changing the UMD module to ES6?

Whether it is feasible with Nuxt.js for changing the module?

I badly in need of this information.
